src/components/BaseButton/index.tsx

```diff
@@ -68,15 +69,15 @@ const StyledButton = styled.button<ButtonProps>`
     background-color: transparent;
     border-radius: ${radius.md};
     opacity: 0.6;
-    box-shadow: 0 0 0 ${space[4]}
-      ${p =>
-        p.variant === ButtonVariant.success
-          ? color.success
-          : p.variant === ButtonVariant.danger
-          ? color.failure
-          : p.variant === ButtonVariant.warning
-          ? color.warning
-          : color.info};
+    box-shadow: 0 0 0 ${space[4]};
+    ${p =>
```

Comment:
Can we rename this to `props` or destruct `variant`? 😅

Reply:
fixed 🙏